manifests/plugin/abrt.pp

```diff
@@ -0,0 +1,28 @@
+# = Foreman Proxy Abrt plugin
+#
+# This class installs abrt plugin
+#
+# === Parameters:
+#
+# $group::        group owner of the configuration file
+#
+# $enabled::      enables/disables the plugin
+#
+#
+class foreman_proxy::plugin::abrt (
+  $enabled  = $::foreman_proxy::plugin::abrt::params::enabled,
```

Comment:
I think we'll need additional parameters here, as the config file contains lots more options: https://github.com/theforeman/smart_proxy_abrt/blob/master/settings.d/abrt.yml.example